### DISCUSSION TITLE
Title

### DISCUSSION THREAD

**Discussion by dosjanaralbaev** [community/community#57933] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

hello

**Comment by dosjanaralbaev** [comment]
hi

**Reply by perk-bevis** [reply]
hi you :))

**Comment by queenofcorgis** [comment]
👋 Welcome to the GitHub Community on Discussions, @dosjanaralbaev  Happy to have you here 🙂

To help keep our space clutter-free, we recommend introducing yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204) and checking out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776). This post will be removed within 24-48 hours to keep our community nice and tidy 🧹.
